### ISSUE TITLE
Invalid Country/City values crashed the processor on devnet

### ISSUE THREAD

**Issue by Mahmoud-Emad** [threefoldtech/tfchain_graphql#143]
### Description

While testing node creation via the Polkadot UI, I encountered an unexpected behavior. Specifically, when setting the values of `Country` and `City` as random numbers, it resulted in a processor crash.

### Related Issue

This issue has been documented and can be tracked [here](https://github.com/threefoldtech/tf_operations/issues/2086).

#### Node details 

```
{
  version: 5
  id: 143
  farmId: 4,413
  twinId: 1,214
  resources: {
    hru: 1,073,741,824,000
    sru: 1,610,612,736,000
    cru: 1
    mru: 1,073,741,824,000
  }
  location: {
    city: 0x4413000150
    country: 0x4413000150
    latitude: 31.8105
    longitude: 31.8105
  }
  publicConfig: null
  created: 1,702,558,662
  farmingPolicyId: 1
  interfaces: []
  certification: Diy
  secureBoot: false
  virtualized: false
  serialNumber: null
  connectionPrice: 80
}

```

**Comment by MarioBassem** [comment]
i added a check to detect invalid country/city values, still to be tested

**Comment by coesensbert** [comment]
> i added a check to detect invalid country/city values, still to be tested

Where is this fix documented? So we can apply to other backends

**Comment by MarioBassem** [comment]
the issue with the processor was that  the country/city string was a hexadecimal encoded invalid utf-8 value, which postgres couldn't store as text.
the fix was to replace these invalid values with empty strings, a temporary solution until invalid values are validated against in the tfchain.

**Comment by MarioBassem** [comment]
the fix in #144 didn't work because the input started with a valid byte 'D'
![image](https://github.com/threefoldtech/tfchain_graphql/assets/26723944/2860da31-f86d-4f0f-9017-dc698de6157d)

updated the fix to search for the invalid data in the entire Uint8Array, and tested it against devnet, the processor was able to process the bad country/city inputs and replaced them with empty strings
